joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -131,6 +108,42 @@ JointTrajectoryController::update()
       registered_joint_cmd_handles_[index]->set_cmd(traj_point_ptr->positions[index]);
     }
 
+    // send feedback
+    if (rt_active_goal_) {
+      auto feedback = std::make_shared<FollowJTrajAction::Feedback>();
+      feedback->header.stamp = lifecycle_node_->now();
+      feedback->joint_names = joint_names_;
+      feedback->actual.time_from_start = rclcpp::Duration(0.0);
+
+      // our current target 
+      auto cur_goal_msg = (*traj_point_active_ptr_)->get_trajectory_msg();
+      if (cur_goal_msg->points.size())
+        feedback->desired = cur_goal_msg->points[0];
+
+      int joint_idx = 0;
+      for (auto& joint_state : registered_joint_state_handles_)
+      {
+        // current frame (only set 1 frame worth of data)
+        feedback->actual.positions.push_back(joint_state->get_position());
+        feedback->actual.velocities.push_back(joint_state->get_velocity());
+        feedback->actual.effort.push_back(joint_state->get_effort());
+
+        // error defined as the difference between current and desired
+        // TODO (ddeng: add angles utility)
```

Comment:
I'm happy to let TODO-s into the repo **as long as** there is also a github issue open and it's referenced in the comment

Reply:
repo found and fixed